refinery/ui/source/js/dashboard/partials/dataset-delete-dialog.html

```diff
@@ -4,17 +4,11 @@ <h3>Delete Dataset</h3>
 <div class="modal-body">
   <span>
     <h5>
-       <i class="fa fa-user"
```

Comment:
Why are we removing this?

Reply:
I don't think there's a reason for the user icon. @mccalluc commented on it.   Did I miss it's purpose?

Reply:
Ah I just looked and I think its redundant to have it. I just wasn't sure!